hyperlane/ethereum/client.go

```diff
@@ -190,47 +206,67 @@ func (c *HyperlaneClient) getISMAddress(ctx context.Context, recipient string) (
 }
 
 func (c *HyperlaneClient) Process(ctx context.Context, domain string, message []byte, metadata []byte) ([]byte, error) {
```

Comment:
slightly refactored the Process function just to pull some logic out into helpers (getting signer and getting address) since that logic is now also needed in QuoteProcessUUSDC